core/src/main/java/io/aiven/kafka/tieredstorage/chunkmanager/ChunkKey.java

```diff
@@ -33,6 +33,14 @@ public ChunkKey(final Uuid uuid, final int chunkId) {
         this.chunkId = chunkId;
     }
 
+    public Uuid getUuid() {
+        return uuid;
+    }
+
+    public int getChunkId() {
+        return chunkId;
+    }
+
```

Comment:
On BytesRange and other classes we used public fields, should we replace these getters with public fields?